pkgs/applications/editors/neovim/neovim-gtk.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, rustPlatform
+, gtk3
+, atk
+, pango
+, cairo
+, gdk_pixbuf
+, glib
+, gobjectIntrospection
+, wrapGAppsHook
+, gnome3
+}:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  name = "neovim-gtk-${version}";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "daa84";
+    repo = "neovim-gtk";
+    rev = "30b7fc87e1e92ac2bb102fd6af6a960fc75017ac";
+    sha256 = "14yy611b6pgy3qay0g1nnwss66xckz0ljw7ixpsqbwi0imjvda27";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+
+  propagatedBuildInputs= [
```

Comment:
This definitely should not be propagated – the `wrapGAppsHook` will take care of it.